.travis.yml

```diff
@@ -9,7 +9,9 @@ jobs:
         - |
           RUN_CLIPPY=true
           rustup component add clippy --toolchain=nightly || RUN_CLIPPY=false
-        - cargo test --all --all-features
+        - cargo test --manifest-path maud/Cargo.toml --features actix-web,iron,rocket
+        - cargo test --manifest-path maud_htmlescape/Cargo.toml
+        - cargo test --manifest-path maud_macros/Cargo.toml
```

Comment:
It's good that you explained this in the commit message -- can you please include that info as a comment here as well?

If there's a relevant Cargo issue then that would be nice to mention too.

Reply:
Can do! I'm not sure there's a relevant issue here, since it seems to be a very intentional choice but I'll see if there's anything relevant!

Reply:
Thanks, looks great!